docs/pages/enroll-resources/kubernetes-access/kubernetes-access.mdx

```diff
@@ -1,31 +1,100 @@
+import iamSvg from "@site/src/components/Icon/svg/awsIdentity.svg";
+import kubernetesSvg from "@site/src/components/Icon/svg/kubernetes2.svg";
 
-- [Introduction to Enrolling Kubernetes Clusters](./introduction.mdx): Learn how Teleport can protect your Kubernetes clusters with RBAC, audit logging, and more.
-- [Enroll a Kubernetes Cluster](./getting-started.mdx): Demonstrates how to enroll a Kubernetes cluster as a resource protected by Teleport.
+{/*vale messaging.protocol-products = NO*/}
+<DocHero
+  title="Enroll your first Kubernetes cluster"
+  youtubeVideoId={
+    // cspell:disable-next-line
+    "rQiloHM2oRQ"
+  }
+  ctaLinks={[
+    {
+      title: "Get started with Kubernetes access",
+      href: "./getting-started/",
+      variant: "secondary",
+      arrow: true,
+    },
+  ]}
+  links={[
+    {
+      title: "Kubernetes Access overview",
+      description: "Securely access Kubernetes clusters with SSO, RBAC and audit logging.",
+      href: "./introduction/",
+      icon: noteSvg,
+    }, 
+    {
+      title: "Troubleshooting",
+      description: "Common Kubernetes access issues and their solutions.",
+      href: "./troubleshooting/",
+      icon: questionSvg,
+    },
+  ]}
+>
+{/*vale messaging.protocol-products = YES*/}
+Log in with SSO and securely access enrolled Kubernetes clusters. Manage access within clusters with granular RBAC down to specific cluster resources, and achieve compliance needs with session recordings and detailed audit logs.
+</DocHero>
 
-## Guides
+<Resources
+  title="Access controls"
+  variant="doc"
+  desktopColumnsCount={2}
+  resources={[
+    {
+      title: "RBAC for Kubernetes",
+      description: "Learn how Teleport's Kubernetes Service intercepts API requests and enforces access with Teleport roles.",
+      iconComponent: identificationBadgeSvg,
+      href: "./controls/"
+    },
+    {
+      title: "Configure and set up",
+      description: "Set up Teleport roles to manage Kubernetes access with a hands-on example using a local Kubernetes cluster.",
+      iconComponent: listChecksSvg,
+      href: "./manage-access/"
+    },
+  ]}
+/>
 
-- [Registering Kubernetes Clusters with Teleport (section)](./register-clusters/): How to manually add a Kubernetes cluster to Teleport after creating it.
-- [Setting Up Access Controls for Kubernetes](./manage-access.mdx): How to configure Teleport roles to access clusters, groups, users, and resources in Kubernetes.
-
-## Configuration & management
-
-- [Access Controls](./controls.mdx): How the Teleport Kubernetes Service applies RBAC to manage access to Kubernetes
-
-## Troubleshooting & support
-
-- [FAQ](./faq.mdx): Frequently asked questions about
-  protecting Kubernetes clusters with Teleport.
-- [Troubleshooting](./troubleshooting.mdx): Troubleshooting
-  common issues with protecting Kubernetes clusters with Teleport.
+<Resources
+  title="Kubernetes cluster registration with Teleport"
```

Comment:
This list misses the our biggest deployment case, installing an agent into the kubernetes cluster